litmus-portal/frontend/src/pages/MyHub/index.tsx

```diff
@@ -191,7 +206,7 @@ const MyHub = () => {
               ) : (
                 //  No GitHub connected div
                 <div className={classes.noGithubAccount}>
-                  <img src="/icons/myhub-crossed.svg" alt="myhub" />
+                  <img src="./icons/myhub-crossed.svg" alt="myhub" />
```

Comment:
In some places, the dot is added and in some places, it is removed. PTAL @amityt